src/webui/www/private/confirmdeletion.html

```diff
@@ -45,9 +45,9 @@
     <br />
 
     <p>&nbsp;&nbsp;QBT_TR(Are you sure you want to delete the selected torrents from the transfer list?)QBT_TR[CONTEXT=HttpServer]</p>
-    &nbsp;&nbsp;&nbsp;&nbsp;<input type="checkbox" id="deleteFromDiskCB" /> <label for="deleteFromDiskCB"><i>QBT_TR(Also delete the files on the hard disk)QBT_TR[CONTEXT=confirmDeletionDlg]</i></label><br /><br />
+    &nbsp;&nbsp;&nbsp;&nbsp;<input type="checkbox" id="deleteFromDiskCB" /> <label for="deleteFromDiskCB"><i>QBT_TR(Also permanently delete the files)QBT_TR[CONTEXT=confirmDeletionDlg]</i></label><br /><br />
```

Comment:
Now `Also` looks redundant.
```suggestion
    &nbsp;&nbsp;&nbsp;&nbsp;<input type="checkbox" id="deleteFromDiskCB" /> <label for="deleteFromDiskCB"><i>QBT_TR(Permanently delete the files)QBT_TR[CONTEXT=confirmDeletionDlg]</i></label><br /><br />
```

Reply:
I don't think it's redundant.

Reply:
>I don't think it's redundant.

Especially now with the addition of "permanently" because it could be interpreted as "If i don't tick the box it will send files to the recycle bin".

Reply:
> it could be interpreted as "If i don't tick the box it will send files to the recycle bin".

IMO, `Also` explicitly indicate that otherwise (i.e. if it is unchecked) the files won't be deleted at all (neither to the trash nor permanently).